test/support/html_assertions.rb

```diff
@@ -46,4 +46,8 @@ def assert_equivalent_html(expected, actual)
   def assert_has_link(expected_text, expected_href, html_fragment)
     assert html_fragment.at_xpath(".//a[.='#{expected_text}'][@href='#{expected_href}']").present?, "Expected\n#{html_fragment.to_s} to include a link with text \"#{expected_text}\" and href \"#{expected_href}\"."
   end
+
+  def assert_has_meta_tag(name, content)
+    assert_select %Q!meta[name="#{name}"][content="#{content}"]!
```

Comment:
Using `{}` is more common as characters for `%` constructions.  Balanced brackets of all kinds are handled.

`%` also defaults to `%Q`:

```
2.1.4 :001 > %Q{a#{123}}
 => "a123" 
2.1.4 :002 > %{a#{123}}
 => "a123" 
```